docs/pages/eas-update/getting-started.md

```diff
@@ -34,25 +34,26 @@ Install EAS and Expo CLIs by running:
 
 Create a project by running:
 
-<Terminal cmd={['$ npx create-expo-app']} />
+<Terminal cmd={['$ npx create-expo-app && ']} />
 
 ## Configure your project
 
-1. Install the latest `expo-updates` library with:
+To configure your project, run the following commands in the order they are specified:
 
-   <Terminal cmd={['$ npx expo install expo-updates']} />
-
-2. Initialize your project with EAS Update:
-
-   <Terminal cmd={['$ eas update:configure']} />
-
-3. To set up the configuration file for builds, run:
-
-   <Terminal cmd={['$ eas build:configure']} />
+<Terminal cmd={[
+'# Install the latest `expo-updates` library',
+'$ npx expo install expo-updates',
+'',
+'# Initialize your project with EAS Update',
+'$ eas update:configure',
+'',
+'# Set up the configuration file for builds',
+'$ eas build:configure',
+]} cmdCopy="expo install expo-updates && eas update:configure && eas build:configure" />
 
-This command will create a file named **eas.json**.
+After running these commands, **eas.json** is created in the root directory of your project.
```

Comment:
Something seems off to me in this rewritten sentence. What do you think about:

```suggestion
After running these commands, **eas.json** file should be present in the root directory of your project.
```

or

```suggestion
After running these commands, **eas.json** file will be created in the root directory of your project.
```